src/sentry/services/hybrid_cloud/rpc.py

```diff
@@ -466,7 +467,8 @@ def create_delegation(cls, use_test_client: bool | None = None) -> DelegatingRpc
         }
         service = DelegatingRpcService(cls, constructors, cls._signatures)
         _global_service_registry[cls.key] = service
-        return service
+        # this returns a proxy which simulates the given class
+        return service  # type: ignore[return-value]
```

Comment:
I thought we were supposed to avoid using `type :ignore`, if our options are to use `cast()` and `type: ignore` is it preferable to choose a small number of ignores over `cast()`?

Reply:
`cast` is worse than `# type: ignore` as it has runtime overhead -- ideally we wouldn't use either and we should make these typesafe.  a little bummed that there's so many typing hacks in HC land at the moment given it's brand new but we can still fix those later on